.travis.yml

```diff
@@ -83,8 +83,14 @@ jobs:
     include:
         - stage: deploy
           if: "tag =~ ^v[0-9]+\\.[0-9]+\\.[0-9]+(a|b|rc)?[0-9]*$"
-          services: ignore
-          addons: skip
+          services: postgresql
```

Comment:
again, on readthedocs this is also not available so it should be possible to do this without a database